plugins/nf-co2footprint/src/main/nextflow/co2footprint/CO2FootprintConfig.groovy

```diff
@@ -25,14 +26,18 @@ class CO2FootprintConfig {
 
     final private String  file
     final private String  summaryFile
+    final private Double  pue // PUE: power usage effectiveness (efficiency coefficient of the data centre)
```

Comment:
Thanks, I wasn't sure either. But from what I see it is fine, it is done here https://www.oracle.com/java/technologies/javase/codeconventions-comments.html and also is done in Nextflow code regularly. 
I do it often when it is only referring to individual variables, because I find it easier to read, in particular if there are various variable listed after each other. Or is there another reason?

But  the page above also says "// shouldn't be used on consecutive multiple lines for text comments", which I did at other parts of the code. I will check this :D

Reply:
Do you see it as a comment in your code editor? I don't think the problem is using `//`, but in this particular line is not rendered as it should

Reply:
Ah I see, weird, I don't know why, it looks as other `//` comments in my editor